core/src/main/java/com/ibm/wala/ipa/callgraph/impl/ExplicitCallGraph.java

```diff
@@ -226,10 +226,6 @@ protected boolean addTarget(int pc, CGNode tNode) {
       }
     }
 
-    /**
```

Comment:
for some reason this causes a javadoc error on 23; i can't understand why.  it's a useless comment anyway so i just deleted it